pkg/azuredisk/azuredisk_v2.go

```diff
@@ -461,7 +465,7 @@ func (d *DriverV2) registerAzDriverNodeOrDie(ctx context.Context) {
 
 	nodeSelector := fmt.Sprintf("metadata.name=%s", d.NodeID)
 
-	azdiskInformer := azdiskinformers.NewSharedInformerFactoryWithOptions(
+	azdiskInformerFactory := azdiskinformers.NewSharedInformerFactoryWithOptions(
```

Comment:
Since this method starts a member informer of `DriverV2`, I am wondering if we should refactor and start it at the beginning of driver setup with the rest of the informers if `d.config.NodeConfig.Enabled`

Reply:
I don't include it to this refactoring for now. Since this informer has an additional need for `Lister()` method [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/10e86f03596907ec945131c1eb4d17bc9e32b74f/pkg/azuredisk/azuredisk_v2.go#L561), I don't see many benefits to combine it with other informers. We could reconsider if we have a better idea.